File: 7/to_lower_case.go
package main

import "fmt"

func toLowerCase(str string) string {
	res := make([]rune, 0)
	for _, c := range str {
		if c >= 65 && c <= 90 {
			res = append(res, c+32)
		} else {
			res = append(res, c)
		}
	}
	return string(res)
}

func main() {
	fmt.Println(toLowerCase("Hello"))
}

File: 7/to_lower_case.py
class Solution:
    def toLowerCase(self, str: str) -> str:
        res = []
        for c in str:
            if 65 <= ord(c) <= 90:
                res.append(chr(ord(c) + 32))
            else:
                res.append(c)
        return ''.join(res)
